migration_toolkit_for_containers/index.adoc

```diff
@@ -0,0 +1,68 @@
+== Upgrading MTC
+xref:../migration_toolkit_for_containers/upgrading-mtc.adoc#upgrading-mtc[Upgrade MTC]: You can upgrade the MTC by using Operator Lifecycle Manager.
+
+[discrete]
+[id="mtc-overview-mtc-checklists"]
+== Reviewing MTC checklists
+xref:../migration_toolkit_for_containers/premigration-checklists-mtc.adoc#premigration-checklists-mtc[MTC checklists]: Before you migrate your application workloads with the Migration Toolkit for Containers (MTC), you can review the premigration checklists.
+
+[discrete]
+[id="mtc-overview-migrate-mtc-applications"]
+== Migrating applications
+xref:../migration_toolkit_for_containers/migrating-applications-with-mtc.adoc#migrating-applications-with-mtc[Migrate applications]:  You can migrate your applications by using the Migration Toolkit for Containers (MTC) xref:../migration_toolkit_for_containers/migrating-applications-with-mtc.adoc#migrating-applications-mtc-web-console_migrating-applications-with-mtc[web console] or xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migrating-applications-cli_advanced-migration-options-mtc[the command line].
+
+[discrete]
+[id="mtc-overview-advanced-migration-options"]
+== Reviewing advanced migration options
+xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#advanced-migration-options-mtc[Advanced migration options]: You can automate your migrations and modify the `MigPlan` and `MigrationController` custom resources in order to perform large-scale migrations and to improve performance. You can check the following items:
+
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-creating-registry-route-for-dim_advanced-migration-options-mtc[Create a registry route for direct image migration]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-configuring-proxies_advanced-migration-options-mtc[Configuring proxies]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-migrating-applications-api_advanced-migration-options-mtc[Migrate an application by using the MTC API]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-state-migration-cli_advanced-migration-options-mtc[Running a state migration]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-hooks_advanced-migration-options-mtc[Creating migration hooks]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-plan-options_advanced-migration-options-mtc[Options for migrating resources]
+* xref:../migration_toolkit_for_containers/advanced-migration-options-mtc.adoc#migration-controller-options_advanced-migration-options-mtc[Migration controller options for improving performance]
+
+[discrete]
+[id="mtc-overview-troubleshooting-mtc"]
+== Troubleshooting MTC
+Troubleshooting MTC: By using logs and debugging tools you can perform the following tasks:
```

Comment:
@xenolinux With my previous comment about removing the xref on "Troubleshooting MTC:", I meant for you to remove the text "Troubleshooting MTC:" as well. (Not just remove the xref).  For example:

```
[discrete]
[id="mtc-overview-troubleshooting-mtc"]
== Troubleshooting MTC
By using logs and debugging tools you can perform the following tasks:
```

And then also to do the same thing for all of the above sections as well. Sorry I wasn't clearer in my comment.